### PR TITLE
Instructions on how to change scale for border-auto-scale presets

### PR DESCRIPTION
**Pull request by nfp0**
This should help users on how to change scale for presets using the new `border-auto-scale.slang` shader.

### CODE REVIEW

**Comment by Tatsuya79** [comment]
It's not necessarily shader #1 scale. "the border pass scale" or something like that.

**Comment by nfp0** [comment]
@Tatsuya79 You're right. It might be any shader nr.

I was scratching my head on how to write it in a way that a user could grasp, but it's not easy to do with such a short character limit, so I had to abbreviate it like this. You think it's perceptible now?

Another problem is that changing scales on the wrong shader passes still scales the picture, but screws the shader quality. So to avoid the user changing the scale on the wrong pass, I've also cleanup up some unnecessary explicit scaling values in the presets which were making them show up as "1X" scale on the shader passes menu. Now the only pass that has "nX" values in the menu is the pass that the user must interact with to change scale. I hope this drives them to the right place.

Please confirm if the changes I made have no impact on the presets at all.

File: handheld/console-border/shader-files/border-auto-scale.slang [hunk 1]
```diff
@@ -8,6 +8,7 @@ layout(push_constant) uniform Push
 	float border_texture_scale;
 } params;
 
+#pragma parameter bogus_scale_instructions "[SCREEN SIZE] To change: Previous menu->LCD shader scale->Apply" 0.0 0.0 1.0 1.0
```
Comment (Tatsuya79) [inline handheld/console-border/shader-files/border-auto-scale.slang:11]: Maybe it's easier to understand with "[SCREEN SIZE] In previous menu->Change LCD shader scale->Apply"

Comment (nfp0) [reply handheld/console-border/shader-files/border-auto-scale.slang]: Yeah, that sounds better.
Pushed. How does this sound now?

Comment (Tatsuya79) [reply handheld/console-border/shader-files/border-auto-scale.slang]: It should be good yes, can't do much more than that. 👍

**Comment by hizzlekizzle** [comment]
Is this something I should merge? Or is there more that needs doing?

**Comment by nfp0** [comment]
I believe so.
The commit I did just now just changes the order of the instructions parameter for better visibility.

**Pull request merged**